zstacklib/zstacklib/utils/qga.py

```
@@ -56,6 +56,10 @@ def decode_with_fallback(encoded_bytes):
             continue
     raise UnicodeDecodeError("Unable to decode bytes using provided encodings")
 
+def merge_cmd(cmd_str, args):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)